.test.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
```

Comment:
Can you also provide the `.travis.xml` file? The idea is that it will call this file on every commit, right? 
Once we have it, we'll take care of integration.